tests/manager.spec.ts

```diff
@@ -101,7 +101,7 @@ describe('HuddlyUsbDeviceManager', () => {
       });
 
       it('should emit USB_DETACH with serial', () => {
-        const detachPromise = new Promise((resolve) => {
+        const detachPromise = new Promise((resolve: any) => {
```

Comment:
Whats up with all the changes on the code?

Reply:
Seems like typescript got a bit stricter

Reply:
👍🏼